src/pages/test-report/components/manoeuvres-popover/manoeuvres-popover.html

```diff
@@ -1,13 +1,14 @@
 <ng-container *ngIf="{
-  manoeuvre: manoeuvres$ | async
+  manoeuvres: manoeuvres$ | async
 } as asyncData">
   <ion-grid class="popover-content">
     <ion-row>
       <ion-col col-36>
         <ion-row>
           <ion-col>
             <input type="radio" name="manoeuvres" 
-            [checked]="asyncData.manoeuvre.selectedReverseRight" 
+            [attr.disabled]="(shouldManoeuvreDisable('reverseRight') | async) ? 'disabled' : null"
```

Comment:
Can you bind `[disabled]` without `attr.` prefix? Think we've done it elsewhere.